src/Sulu/Component/Persistence/EventSubscriber/ORM/UserBlameSubscriber.php

```diff
@@ -122,56 +122,45 @@ public function onFlush(OnFlushEventArgs $event)
             return;
         }
 
-        $user = null;
+        $user = $this->getUser($token);
+
+        // if no sulu user, do nothing
+        if (!$user instanceof UserInterface) {
+            return;
+        }
+
+        $this->handleUserBlame($event, $user, true);
+        $this->handleUserBlame($event, $user, false);
```

Comment:
I don't really see why we have to call that method twice. When I look at the code it looks to me like the changer is also set in the first call.

Reply:
there could be insertions and updates in the same session